src/app/retrospective/retrospective-feedback/retrospective-feedback.component.ts

```diff
@@ -301,19 +322,23 @@ export class RetrospectiveFeedbackComponent implements OnInit, OnChanges, OnDest
         }
     }
 
-    getTeamMemberOptions(teamMembers) {
+    getTeamMemberOptions() {
+        let teamMembers = this.teamMembers;
         teamMembers = _.map(teamMembers, (data: any) => {
             return {
                 id: _.parseInt(data.ID),
                 value: (data.FirstName + ' ' + data.LastName).trim(),
             };
         });
-        return [{ id: null, value: 'None' }, ...teamMembers];
+        return teamMembers;
     }
 
-    revertCellValue(params) {
+    revertCellValue(params, ...assigneeIDPrevValue) {
```

Comment:
We should use a generic name for the input variable as:
```suggestion
    revertCellValue(params, ...revertParams) {
```
and use it with proper comments for each case.